examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart

```diff
@@ -146,6 +146,7 @@ class CupertinoDemoTab1 extends StatelessWidget {
   Widget build(BuildContext context) {
     return CupertinoPageScaffold(
       child: CustomScrollView(
+        semanticChildCount: 50,
```

Comment:
nit: would it make sense to have a const `childCount` that can be passed in here and below for `childCount` so when one of them gets updated both get updated?